WordPress/Classes/Services/AccountService.h

```diff
@@ -81,6 +81,14 @@ extern NSString *const WPAccountEmailAndDefaultBlogUpdatedNotification;
 - (WPAccount *)createOrUpdateAccountWithUsername:(NSString *)username
                                        authToken:(NSString *)authToken;
 
+/**
+ Retrieves all the intances of Account retrieve the complete information from the backend if necessary
+ 
+ @param completion the block where all the completed WPAccounts in the array will be returned
+ */
+typedef void (^WPAccountRetrieveCompletion)(NSArray* accounts);
```

Comment:
@koke  I agree I was thinking a head in case I needed again, but I didn't, I'll remove it